assets/src/modules/Lizmap.js

```diff
@@ -82,7 +82,7 @@ export default class Lizmap {
                 // child which could be a layer or a group.
                 let wmsLayer = wmsCapabilities.Capability.Layer;
                 while (wmsLayer.BoundingBox === undefined) {
-                    for (const wmsChildLayer of wmsLayer.Layer) {
+                    for (const wmsChildLayer of wmsLayer.Layer || {}) {
```

Comment:
```suggestion
                    // breaking while before the loop because wmsLayer.Layer is not iterable
                    if (wmsLayer.Layer === undefined) {
                        break;
                    }
                    for (const wmsChildLayer of wmsLayer.Layer) {
```